.circleci/workflows.yml

```diff
@@ -33,7 +33,7 @@ parameters:
 executors:
   ubuntu-machine-executor:
     machine:
-      image: ubuntu-2004:202111-02
+      image: ubuntu-2004:current
```

Comment:
While we could update this to a specific newer supported image version to avoid any possibility of breaking changes being introduced, IMO having the overall Ubuntu major version locked should provide reasonable stability and this will save us having to do such updates quite so often as CircleCI deprecates older specific version images over time.